src/uwtools/drivers/driver.py

```diff
@@ -33,8 +33,6 @@
 from uwtools.utils.file import writable
 from uwtools.utils.processing import run_shell_cmd
 
-OutputT = dict[str, Union[str, list[str]]]
```

Comment:
Unrelated simplification, see also the updated return types of the `output()` methods.